### PR TITLE
IDEMPIERE-6275 Fix row presentation after first time find modal is closed (User preference option - View find result and ZK_GRID_AFTER_FIND)

### PR DESCRIPTION
**Pull request by MilosR**
https://idempiere.atlassian.net/browse/IDEMPIERE-6275

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

**Review by hengsin — CHANGES_REQUESTED** [review]
see comments in ticket.

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/IADTabpanel.java [hunk 1]
```diff
@@ -116,6 +116,11 @@ public interface IADTabpanel extends Component, Evaluatee {
 	 */
 	public void switchRowPresentation();
 
+	/**
+	 * Setup Row presentation - switch to grid view if configured
+	 */
+	public void setupRowPresentation();
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/IADTabpanel.java:122]: please rename this to onAfterFind()

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/ADTabpanel.java [hunk 2]
```diff
@@ -976,9 +978,6 @@ public String get_ValueAsString(String variableName) {
     			Events.echoEvent(ON_DEFER_SET_SELECTED_NODE, this, null);
     		}        	
         }
-
-        if (!update && !gridTab.isSingleRow() && !isGridView())
-        	switchRowPresentation();
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/ADTabpanel.java:981]: this should not be removed.

**Review by hengsin — CHANGES_REQUESTED** [review]

**Comment by MilosR** [comment]
@hengsin Method renamed and logic for switchRowPresentation is restored

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**